standard_schema/prerelease/HED8.3.0.mediawiki

```diff
@@ -505,30 +505,30 @@ Each term in this vocabulary has a human-readable description and may include ad
+***** Eye-blink-artifact <nowiki>[Artifact from eye blinking. In EEG, Fp1/Fp2 electrodes become electropositive with eye closure because the cornea is positively charged causing a negative deflection in Fp1/Fp2. If the eye blink is unilateral, consider prosthetic eye.]</nowiki>
+***** Eye-movement-artifact <nowiki>[Eye movements can cause artifacts on recordings. The charge of the eye can especially cause artifacts in electrophysiology data.]</nowiki>
+****** Horizontal-eye-movement-artifact <nowiki>[Artifact from moving eyes left-to-right and right-to-left. In EEG, there is an upward deflection in the Fp2-F8 derivation, when the eyes move to the right side. In this case F8 becomes more positive and therefore. When the eyes move to the left, F7 becomes more positive and there is an upward deflection in the Fp1-F7 derivation.]</nowiki>
+****** Vertical-eye-movement-artifact <nowiki>[Artifact from moving eyes up and down. In EEG, this causes positive potentials (50-100 micro V) with bi-frontal distribution, maximum at Fp1 and Fp2, when the eyeball rotated upward. The downward rotation of the eyeball was associated with the negative deflection. The time course of the deflections is similar to the time course of the eyeball movement.]</nowiki>
+****** Slow-eye-movement-artifact <nowiki>[Artifacts originating from slow, rolling eye-movements, seen during drowsiness.]</nowiki>
+****** Nystagmus-artifact <nowiki>[Artifact from repetitive, uncontrolled movements of the eyes (nystagmus).]</nowiki>
+**** Chewing-artifact <nowiki>[Artifact from moving the jaw in a chewing motion.]</nowiki>
+**** Sucking-artifact <nowiki>[Artifact from sucking, typically seen in very young cases.]</nowiki>
+**** Tongue-movement-artifact <nowiki>[Artifact from tongue movement (Glossokinetic). The tongue functions as a dipole, with the tip negative with respect to the base. In EEG, the artifact produced by the tongue has a broad potential field that drops from frontal to occipital areas, although it is less steep than that produced by eye movement artifacts. The amplitude of the potentials is greater inferiorly than in parasagittal regions; the frequency is variable but usually in the delta range. Chewing and sucking can produce similar artifacts.]</nowiki>
+**** Rocking-patting-artifact <nowiki>[Quasi-rhythmical artifacts in recordings most commonly seen in infants. Typically caused by a caregiver rocking or patting the infant.]</nowiki>
+**** Movement-artifact <nowiki>[Artifact in the measured data generated by motion of the subject.]</nowiki>
+**** Respiration-artifact <nowiki>[Artifact from breathing.]</nowiki>
 **** Pulse-artifact <nowiki>[Artifact from a pulsating blood vessel near a measurement site, cardiobalistic artfact.]</nowiki>
 **** ECG-artifact <nowiki>[Artifact from pulsation of the heart, timelocked to QRS complex.]</nowiki>

```

Comment:
The ECG and pulse artifacts may not be very clear. Is the `pulse artifact` a **mechanical** artifact due to the pulsating movement of the blood vessel and the `ECG artifact` an **electrical** artifact?

Reply:
I corrected the ecg and pulse artifact with suggested changes